projects/backend/utils/crossword.ts

```diff
@@ -56,17 +56,22 @@ type CrosswordArticleOverrides = Pick<
 > &
     Pick<CrosswordArticle, 'crossword'>
 
-const shouldHaveSolution = (type: CrosswordType) =>
-    ![CrosswordType.EVERYMAN, CrosswordType.PRIZE].includes(type)
```

Comment:
I'd be more comfortable if we always skip solutions for crosswords of type `CrosswordType.PRIZE`. I'm guessing they'd would always have a `dateSolutionAvailable` in the future, but just in case, I'd prefer keeping this logic.